projects/Amlogic/packages/device-trees-amlogic/package.mk

```diff
@@ -0,0 +1,49 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2016-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="device-trees-amlogic"
+PKG_VERSION="68267ba"
+PKG_SHA256=""
```

Comment:
sha256 missing :)

Reply:
This is WIP, I will add SHA256 when package version is ready to go. For now PKG_VERSION only is provided for easier updates (for me) - and it works.